includes/admin/post-types/meta-boxes/class.llms.meta.box.achievement.sync.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
I didn't want to mess with this on roccos initial work but since the autoloader will now pick this up with either space or dash delimiter let's change the filename to be dashes (as we intend new files to be)